gulpfile.js

```diff
@@ -18,7 +18,7 @@ var envVars = {
 
 /* jshint camelcase: false */
 gulp.task('style', function () {
-  gulp.src([paths.sourceFiles, paths.testFiles, paths.gulpFile])
+  return gulp.src([paths.sourceFiles, paths.testFiles, paths.gulpFile])
```

Comment:
The reason it wasn't do it before was the lack of `return` on both `lint` and `style`.